homeassistant/components/lametric/quality_scale.yaml

```diff
@@ -0,0 +1,75 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
How does `test_cloud_import_updates_existing_entry` work? Ideally we don't use the config flow to reconfigure a device, we rather use the reconfigure flow for that

Reply:
I think `test_manual_updates_existing_entry` should be a reconfigure flow for sure

Reply:
These re-import auth from the cloud. It is the only function of this part. There is nothing to re-configure, as it doesn't exist beyond the point of the configuration flow.